extensions/VisualEditor/wikia/modules/ve/ce/ve.ce.WikiaVideoNode.js

```diff
@@ -72,7 +72,17 @@ ve.ce.WikiaVideoNode.static.getSizeString = function ( width ) {
  * @returns {jQuery}
  */
 ve.ce.WikiaVideoNode.prototype.createPlayButton = function () {
-	return this.$( '<span>' ).addClass( 'play-circle ve-no-shield' );
+	return this.$( '<span>' ).addClass( 'thumbnail-play-icon-container ve-no-shield' ).html(
+		'<svg class="thumbnail-play-icon" viewBox="0 0 180 180" width="100%" height="100%">' +
```

Comment:
@hakubo we should take care about https://wikia-inc.atlassian.net/browse/XW-3926 sooner than later to not have to do such ugly things like pasting svg markup into js files